packages/grafana-ui/src/components/Logs/LogDetailsRow.test.tsx

```diff
@@ -14,6 +14,9 @@ const setup = (propOverrides?: Partial<Props>) => {
     getStats: () => null,
```

Comment:
Just a nit: I know this test file is still using enzyme, would it make sense to write the new ones using testing-library instead?

Reply:
Yes, very good point. Done in https://github.com/grafana/grafana/pull/34125/commits/69063bb243ec262e20f7bedaabb45e5b04ac8b55